### ISSUE TITLE
SET clause doesn't work with list properties

### ISSUE THREAD

**Issue by pdpotter** [apache/incubator-age#44]
It is possible to create a node with a list property:
```
SELECT * FROM cypher('testgraph', $$CREATE (v:Test {id: 1, a: ['a', 'b']}) return v$$) as (a agtype);
                                             a                                              
---------------------------------------------------------------------------------------------
 {"id": 1688849860263973, "label": "Test", "properties": {"a": ["a", "b"], "id": 1}}::vertex
(1 row)
```
It seems to be impossible to update existing nodes with list properties:
```
SELECT * FROM cypher('testgraph', $$MATCH (ve:Test {id: 1}) SET ve.a = ['c', 'd'] return ve$$) as (a agtype);
                                          a                                           
--------------------------------------------------------------------------------------
 {"id": 1688849860263974, "label": "Test", "properties": {"a": "c", "id": 1}}::vertex
(1 row)

```
It also seems to be impossible to add new list properties to existing nodes:
```
SELECT * FROM cypher('testgraph', $$MATCH (ve:Test {id: 1}) SET ve.b = ['e', 'f'] return ve$$) as (a agtype);
                                               a                                                
------------------------------------------------------------------------------------------------
 {"id": 1688849860263974, "label": "Test", "properties": {"a": "c", "b": "e", "id": 1}}::vertex
(1 row)
```

**Comment by kysmou** [comment]
This part seems to need to be fixed.
The same occurs even when I tested it, and it seems that only the first value is updated and saved for list data.

SELECT * FROM cypher('graph_name', $$MATCH (ve:Person {id: 1}) SET ve.title=['1','2'] return ve $$) as (a agtype);
                                a                                                  
-----------------------------------------------------------------------------------------------------
{"id": 844424930131972, "label": "Person", "properties": {"a": "c", "id": 1, "title": "1"}}::vertex
(1 row)

**Comment by LinuxJedi** [comment]
Doh! I just spent far too long trying to figure out why this wasn't working in our application...

**Comment by pdpotter** [comment]
This issue is now fixed on master (by commit ab70781b4b311922f14f2bcb1b2144f1df9a9da1).

This is great, thanks!
